composer.json

```diff
@@ -15,12 +15,12 @@
     "require": {
         "php": "^7.2 || ^8.0",
         "matthiasnoback/symfony-config-test": "^4.0.1",
-        "symfony/dependency-injection": "^3.4 || ^4.4 || ^5.0",
-        "symfony/config": "^3.4 || ^4.4 || ^5.0",
-        "symfony/yaml": "^3.4 || ^4.4 || ^5.0"
+        "symfony/dependency-injection": "^4.4 || ^5.3 || ^6.0",
+        "symfony/config": "^4.4 || ^5.3 || ^6.0",
+        "symfony/yaml": "^4.4 || ^5.3 || ^6.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.0"
+        "phpunit/phpunit": "^8.0 || ^9.0"
```

Comment:
PHPUnit 9 is needed with the Symfony 6 components otherwise there's a broken test:

```sh
The data provider specified for Matthias\SymfonyDependencyInjectionTest\Tests\PhpUnit\ContainerHasParameterConstraintTest::match is invalid.
ParseError: syntax error, unexpected variable "$value"
Tests/PhpUnit/ContainerHasParameterConstraintTest.php:49
Tests/PhpUnit/ContainerHasParameterConstraintTest.php:29
```

The issue comes from trying to create a mock of the container's interface, Symfony 6 has union types and the mock builder in PHPUnit 8 doesn't like those.